### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by MudassirNoor**
Minor updates to README.md and VERSION.md for better readability :).

### CODE REVIEW

File: VERSIONING.md [hunk 1]
```diff
@@ -63,4 +63,4 @@ The command-line API is defined by the documentation for the GUI, which includes
 
 Even in the case of non-breaking changes, as defined above, every effort will be made to avoid negative impact on users.
 
-The included **Experimental GUI** is not subject to the above considerations. It is currently versioned separately from the standard GUI, with a MAJOR component of 0. The **Experimental GUI** may gain and lose features at any time until it becomes (as we eventually intend) the second major version of the GUI.
```
Comment (CharliePoole) [inline VERSIONING.md:66]: The Experimental GUI no longer exists, so let's drop this paragraph.

File: VERSIONING.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-- PATCH version when you making backwards-compatible bug fixes
+- `MAJOR` version when making incompatible API changes.
+- `MINOR` version when adding functionality in a backwards-compatible manner.
+- `PATCH` version when making backwards-compatible bug fixes.
 
 What remains is to define the API or APIs we are protecting from incompatible changes and the degree of change that is needed before something is considered a breaking change.
 
 ## TestCentric APIs
 
-First, let's note that the **TestCentric GUI** is not designed to be used as a library, so there is no ABI or library-like API to worry about. So what other APIs do we need to support?
```
Comment (CharliePoole) [inline VERSIONING.md:13]: ABI is correct... Application Binary Interface as opposed to a (source-level) API. That's why both are mentioned.

**Review by CharliePoole — CHANGES_REQUESTED** [review]
Thanks for doing this. I think it generally improves the text. See two comments for changes I'd like to see.

File: VERSIONING.md (hunk 2 above)
Comment (MudassirNoor) [reply VERSIONING.md:13]: My bad, I will revert that change.

**Review by CharliePoole — APPROVED** [review]
Looks good. Thanks for your work.

**Pull request merged**